script/blog.js

```diff
@@ -0,0 +1,66 @@
+const headOne = document.createElement('h2');
+headOne.textContent = 'Adoptadog Blog';
+document.getElementById("container").appendChild(headOne);
+
+const i = [{
```

Comment:
I like this strategy. One thing that will make your code a little better is more semantic variable names. For example, instead of just using i for the blog post object, perhaps blogPosts would be a better choice.